### ISSUE TITLE
⌨ Disabled outline link buttons have insufficient colour contrast

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#6394]
## ⌨ Accessibility Issue

If a link with the button style is disabled, it reduces the colour contrast lower than the expected threshold (4.5:1).

### WCAG 2.1 Criteria

[Understanding SC 1.4.3: Contrast (Minimum) (Level AA)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

## 🦋 Expected Behaviour

Even when the link buttons are disabled, the foreground should have a contrast ratio of 4.5:1 compared to the background.

## 📋 Steps to Reproduce

1.  `npm run storybook`
2. Navigate to the "link button disabled" story
3. Change the mode to `outline`
4. Run lightouse
5. See the colour contrast errors

## 📸 Screenshot

![Screenshot 2023-04-25 163242](https://user-images.githubusercontent.com/4127998/234396964-58aff973-fce4-478a-bfca-f5d71e01157b.png)

## 🙋‍♀️ Proposed Solution

Possibly instead of reducing opacity when a link is disabled, we could use a lighter variation of the colour to have more control over the computed values? 🤔 

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the issue was addressed.

- [ ] All variations of disabled link buttons meet the minimum colour contrast ratio (4.5:1)

**Comment by esizer** [comment]
This was fixed by the new button/link components
